doc/abstract-logger.md

```diff
@@ -38,7 +38,7 @@ class CustomLogger(AbstractLogger):
 Following is how the custom logger implementation can be injected in the SDK client.
 
 ```python
-client = PaypalserversdkClient(
+client = PaypalServersdkClient(
```

Comment:
Maybe this is a Python convention thing, but this seems like an odd name. I get why it switched the `server` to `Server`, but it still seems like it's doing something kind of weird with the entire token that is our package name.